### DISCUSSION TITLE
Card padding discrepancy between title and description

### DISCUSSION THREAD

**Discussion by jknilans** [RedHat-UX/red-hat-design-system#1556] [General]
Hello, I noticed that the [card component documentation](https://github.com/orgs/RedHat-UX/discussions/categories/general) says that there should be a 16px spacer between the title and description. This seems right, however when I look at the [live demo](https://ux.redhat.com/elements/card/#demos) and [demo variants page](https://ux.redhat.com/elements/card/demo/variants/), there appears to be a total of 48px of spacing applied. 

We implemented the card and noticed that it looks like the 48px spacer is what comes as default, but had to remove the default spacing and use a 16px spacer to get the right padding between title and description.

It looks like what may be cause is that the card body has a `margin-top` of 32px and the `p` tag has a `margin-top` of 16px, which when added together make the 48px spacer.

Can the components to be updated to reflect the 16px spacing that is used on the [Desktop/Mobile style documentation](https://ux.redhat.com/elements/card/style/#desktop) page? Thanks!

**Comment by bennypowers** [comment]
this came up earlier [here](https://github.com/RedHat-UX/red-hat-design-system/issues/1384#issuecomment-1859167903)

perhaps it's time to reconsider adding an `image` slot (or some other better name for it)

cc @marionnegp and @zeroedin 

WDYT, how essential is the image to the layout here? how specific does the use of images in card have to be? do we risk shooting ourselves in the foot if we prescribe a single "golden" use of images in cards? we could begin the process of adding any number of card variants, e.g. inline promo, image heading vertical, image heading horizontal, etc. one potential downside to that approach is that developers may find it distasteful to use card (which ships with code to account for many different variants) if they only need a single variant on the page

**Reply by marionnegp** [reply]
Would we have to define whether an image was horizontal or vertical? Sounds like something the designer or dev using the card should be responsible for.

**Reply by jknilans** [reply]
FWIW, we weren't using an image card.

**Comment by bennypowers** [comment]
This should be addressed in the upcoming clefairy release (#1802 )
